worker/manager/src/mirrors_qa_manager/worker.py

```diff
@@ -93,6 +93,20 @@ def get_host_fpath(self, container_fpath: Path) -> Path:
         """Determine the host path of a path in the container."""
         return self.host_workdir / container_fpath.relative_to(Settings.WORKDIR_FPATH)
 
+    def get_country_codes_from_config_files(self) -> list[str]:
+        """Get the ISO country codes of countries using configuration files.
+
+        Finds all files ending with .conf and applies the following steps:
+        - take the first two letters of the config filename.
+        - add to output list if first two letters of config are valid country codes
+        """
+        country_codes = set()
+        for conf_file in self.base_dir.glob("*.conf"):
+            code = conf_file.name[:2]
```

Comment:
That's a bit fragile. We should use a regex over the `conf_file.stem` (`r"^[a-z{2}-") and ignore if the file does not match